### PR TITLE
Add Parameters endpoints

### PR DESCRIPTION
**Pull request by ayolab**

### CODE REVIEW

File: src/main/resources/application-local.yml [hunk 1]
```diff
@@ -12,12 +12,19 @@ powsybl:
       
 gridsuite:
   services:
-    study-server:
+    -
```
Comment (etiennehomer) [inline src/main/resources/application-local.yml:15]: It was not working anymore with old version of this section ?

File: src/main/java/org/gridsuite/explore/server/ExploreController.java [hunk 2]
```diff
@@ -34,6 +35,8 @@ public class ExploreController {
     // /!\ This query parameter is used by the gateway to control access
     private static final String QUERY_PARAM_PARENT_DIRECTORY_ID = "parentDirectoryUuid";
 
+    private static final String QUERY_PARAM_PARAMETERS_TYPE = "type";
```
Comment (etiennehomer) [inline src/main/java/org/gridsuite/explore/server/ExploreController.java:38]: You can move this in ParametersService

**Comment by SlimaneAmar** [comment]
Maybe it's better to rename `parameters` by `setting` ?

File: src/main/resources/application-local.yml (hunk 1 above)
Comment (SlimaneAmar) [reply src/main/resources/application-local.yml:15]: Why ?

Comment (ayolab) [reply src/main/resources/application-local.yml:15]: so we can get the map of services and get the baseUri of the server that we need to call based on parameters type

File: src/main/java/org/gridsuite/explore/server/services/RemoteServicesProperties.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    public String getServiceUri(String serviceName) {
+        String defaultUri = "http://" + serviceName + "/";
+        return Objects.isNull(services) ? defaultUri : services.stream()
+                .filter(s -> s.getName().equalsIgnoreCase(serviceName))
+                .map(RemoteServicesProperties.Service::getBaseUri)
+                .findFirst()
+                .orElse(defaultUri);
+    }
+
+    public void setServiceUri(String serviceName, String newUri) {
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/explore/server/services/RemoteServicesProperties.java:41]: Remove this method

**Review by SlimaneAmar — APPROVED** [review]

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**